gandalf.js

```diff
@@ -1,24 +1,27 @@
-/**
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.